archaius2-core/src/test/java/com/netflix/archaius/property/PropertyTest.java

```diff
@@ -167,6 +168,43 @@ public void testCollectionTypes() {
         // Test proper handling of unset properties
         Property<Map<CustomType, CustomType>> emptyProperty = factory.getMap("fubar", CustomType.class, CustomType.class);
         Assert.assertNull(emptyProperty.get());
+
+        config.setProperty("fubar", "");
+        Property<List<String>> emptyListProperty = factory.getList("fubar", String.class);
+        Assert.assertEquals(Collections.emptyList(), emptyListProperty.get());
```

Comment:
Since all the default collections are value types, this assertion is really just equivalent to `assertTrue(emptyList.isEmpty())`, isn't it?

Reply:
Yeah, that's equivalent.